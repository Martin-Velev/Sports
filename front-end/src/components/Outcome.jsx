import React from "react";

const Outcome = props => {
  return (
    <div>
      <h5>The score is:</h5>
      {props.outcome.scoreA} : {props.outcome.scoreA}
    </div>
  );
};

export default Outcome;
