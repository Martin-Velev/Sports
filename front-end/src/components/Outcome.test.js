import React from "react";
import renderer from "react-test-renderer";
import Outcome from "../components/Outcome";
import { BrowserRouter as Router } from "react-router-dom";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Router>
        <Outcome outcome={{scoreA: 42, scoreB: 42}}/>
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
