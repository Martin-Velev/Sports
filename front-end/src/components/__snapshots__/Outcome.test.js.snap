// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h5>
    The score is:
  </h5>
  42
   : 
  42
</div>
`;
